notebooks/halos_rendering.ipynb

```diff
@@ -0,0 +1,457 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    kappa_gamma_distribution = halos_lens.get_kappa_gamma_distib(gamma_tot=True, diff=None)
</pre><p>The function should be called as <code>get_kappa_gamma_distribution()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sibirrer/sim-pipeline/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>